pom.xml

```diff
@@ -49,12 +49,6 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
         </dependency>
```

Comment:
Was this a duplicate dependency?

It might be better to put the removal of this into a different pull request--that makes it easier to code review.

Reply:
Okay fixed that.  Will make a new pull request for this